homeassistant/components/ecowitt/sensor.py

```diff
@@ -152,24 +152,28 @@
         native_unit_of_measurement=UnitOfPrecipitationDepth.MILLIMETERS,
         device_class=SensorDeviceClass.PRECIPITATION,
         state_class=SensorStateClass.TOTAL_INCREASING,
+        suggested_display_precision=1,
     ),
     EcoWittSensorTypes.RAIN_COUNT_INCHES: SensorEntityDescription(
         key="RAIN_COUNT_INCHES",
         native_unit_of_measurement=UnitOfPrecipitationDepth.INCHES,
         device_class=SensorDeviceClass.PRECIPITATION,
         state_class=SensorStateClass.TOTAL_INCREASING,
+        suggested_display_precision=3,
```

Comment:
Do we really want 3 decimals? 0.001 inch is 0.025 mm, which seems overly precise.
A hundredth of an inch is 0.25 mm which seems more reasonable.

Note that the user can customize the display precision themselves from the UI.